src/theory/quantifiers/theory_quantifiers.cpp

```diff
@@ -29,12 +29,12 @@
 #include "theory/quantifiers_engine.h"
 #include "theory/valuation.h"
 
-using namespace std;
-using namespace CVC4;
 using namespace CVC4::kind;
```

Comment:
Weren't we trying to keep this explicit as well?

Reply:
No, I like using it for files with heavy use of kind since it shrinks line sizes.

Reply:
OK, sounds good.